src/Connection/ConnectionManager.php

```diff
@@ -67,6 +67,15 @@ private function findConnectionProvider( $id ) {
 			return self::$connectionProviders[$id];
 		}
 
+		// #4150
+		if ( !defined( 'SMW_VERSION' ) ) {
+			$setupCheck = new SetupCheck();
```

Comment:
Have a look the posted change, see `use SMW\SetupCheck`.

Reply:
Thanks for the pointer. I just have too many windows open so I pretty messed :(